File: dirA/populacao.c
# include <stdio.h>
# include <cs50.h>

int main (void)
{
    int inicio, fim, i;
    inicio = 0;
    do
    {
        inicio = get_int ("Inicio: ");
    }
    while (inicio < 9);
    do
    {
        fim = get_int ("Fim: ");
    }
    while (fim < inicio);
    while (inicio < fim){
        inicio = inicio + (inicio/3) - (inicio/4);
        i ++;
    }
    printf (tot)

}

File: Python/populacao.py
inicio = 0
fim = 0
while inicio < 9:
    inicio = int(input('Inicio da população: '))
while fim < inicio:
    fim = int(input('Valor final: '))
ano = 1
mortos = int(inicio / 4)
nasc = int(inicio / 3)
tot = inicio + nasc - mortos
while tot < fim:
    mortos = int(tot/4)
    nasc = int(tot/3)
    tot = tot + nasc - mortos
    ano +=1
print(ano)   